Source/cursor.cpp

```diff
@@ -540,14 +540,16 @@ void CheckCursMove()
 	if (pcursmonst == -1) {
 		if (!flipflag && mx + 1 < MAXDUNX && dPlayer[mx + 1][my] != 0) {
 			int8_t bv = abs(dPlayer[mx + 1][my]) - 1;
-			if (bv != MyPlayerId && Players[bv]._pHitPoints != 0) {
+			Player &player = Players[bv];

```

Comment:
This is going to cause conflicts because in #4754 I've renamed `bv` to `playerId`

Reply:
Yeah, it will be the second one here, defiantly the most painful conflicts, but ok handling it :)